jobs/process_paid_filings/Dockerfile

```diff
@@ -79,12 +81,14 @@ RUN --mount=type=cache,target="$POETRY_CACHE_DIR" \
     --no-interaction --no-ansi
 
 # Running as non-root user:
-USER web
-
-# The following stage is only for production:
-FROM development_build AS production_build
-COPY --chown=web:web . /code
-
-# ENV PYTHONPATH=/opt/app-root/src
-
-CMD [ "python", "-m", "process_paid_filings" ]
+  COPY --chown=web:web . /code
```

Comment:
why did you change indentation?

Reply:
fixed it, copied from the ar-reminder dockerfile and didn't realize the indentation was changed